public/app/features/trails/ActionTabs/LayoutSwitcher.tsx

```diff
@@ -1,21 +1,44 @@
 import { SelectableValue } from '@grafana/data';
-import { SceneComponentProps, sceneGraph, SceneObject, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import {
+  SceneComponentProps,
+  SceneObject,
+  SceneObjectBase,
+  SceneObjectState,
+  SceneObjectUrlSyncConfig,
+  SceneObjectUrlValues,
+  SceneObjectWithUrlSync,
+} from '@grafana/scenes';
 import { Field, RadioButtonGroup } from '@grafana/ui';
 
-import { MetricScene } from '../MetricScene';
 import { reportExploreMetrics } from '../interactions';
-import { TRAIL_BREAKDOWN_VIEW_KEY } from '../shared';
+import { MakeOptional, TRAIL_BREAKDOWN_VIEW_KEY } from '../shared';
 
-import { LayoutType } from './types';
+import { BreakdownLayoutType } from './types';
 
 export interface LayoutSwitcherState extends SceneObjectState {
+  breakdownLayout: BreakdownLayoutType;
   layouts: SceneObject[];
-  options: Array<SelectableValue<LayoutType>>;
+  options: Array<SelectableValue<BreakdownLayoutType>>;
 }
 
-export class LayoutSwitcher extends SceneObjectBase<LayoutSwitcherState> {
-  private getMetricScene() {
-    return sceneGraph.getAncestor(this, MetricScene);
+export class LayoutSwitcher extends SceneObjectBase<LayoutSwitcherState> implements SceneObjectWithUrlSync {
+  protected _urlSync = new SceneObjectUrlSyncConfig(this, { keys: ['breakdownLayout'] });
```

Comment:
this will be a url and persistence breaking change (layout => breakdownLayout) , jus t so you know